compiler/src/iree/compiler/GlobalOptimization/Utils.cpp

```diff
@@ -121,5 +121,67 @@ wrapConsecutiveOpsInDispatchRegion(RewriterBase &rewriter,
   return IREE::Flow::movePrecedingOpsIntoDispatchRegion(rewriter, precedingOps,
                                                         regionOp);
 }
+// Reduce the input value along the reduction dimensions
+Value sumReduceDimensionSubset(ImplicitLocOpBuilder &rewriter, Value val,
+                               Type accETy, ArrayRef<bool> is_reduction) {
```

Comment:
nit: Move the function comment to where it is declared (i.e., Passes.h). Also, lets have a blank line between functions. It usually improves readability.

```suggestion

Value sumReduceDimensionSubset(ImplicitLocOpBuilder &rewriter, Value val,
                               Type accETy, ArrayRef<bool> is_reduction) {
```